editor/package.json

```diff
@@ -183,7 +183,8 @@
     "string-hash": "1.1.3",
     "strip-ansi": "6.0.0",
     "tippy.js": "6.2.6",
-    "typescript": "4.0.5",
+    "twind": "0.16.16",
+    "typescript": "4.1.3",
```

Comment:
Twind required a slightly updated version of TS, which is the cause of some unrelated changes elsewhere. There is a much newer version, but we didn't want to introduce too much noise into this PR, so that is something we should tackle separately.